docs/faststore/docs/getting-started/1-onboarding/starting-the-project.mdx

```diff
@@ -8,6 +8,8 @@ Now that you have an [overview of the app](https://developers.vtex.com/docs/guid
     - Configure a GitHub repository for your project.
     - Generate a live preview of your store.
 
+>⚠ To start a FastStore project, users must have the Owner (Admin Super) role.
```

Comment:
Este perfil de acesso é necessário apenas após a conclusão do fluxo de onboarding no WebOps ou durante o próprio processo? Caso seja a última opção, além desse callout aqui, talvez adicionar uma menção a este requisito na seção [Before you begin](https://developers.vtex.com/docs/guides/faststore/1-onboarding-overview#before-you-begin) da visão geral do WebOps. Isso porque, neste ponto, o usuário já teria passado pelo onboarding.